Apps/WebClient/src/ClientApp/src/components/timeline/encounter.vue

```diff
@@ -65,9 +65,6 @@ export default class EncounterTimelineEntryComponent extends Vue {
                     <b-col>
                         <div class="d-flex flex-row-reverse">
                             <b-btn
```

Comment:
While you’re in the code it would be ideal to add the testids as it would save future work.